sensor_msgs/PointCloud2 colored_cloud 
sensor_msgs/PointCloud2[] segmented_clouds
pcl_msgs/PointIndices[] Indices
